feg/gateway/docker/.env

```diff
@@ -24,3 +24,5 @@ CONFIGS_TEMPLATES_PATH=../../../orc8r/gateway/configs/templates
 
 CERTS_VOLUME=gwcerts
 CONFIGS_VOLUME=gwconfigs
+
+LOG_DRIVER=journald
```

Comment:
For those gateways not running in Helm, I understand for now we have to change this manually, correct?